server/helpers/funtions.js

```diff
@@ -16,3 +16,20 @@ export const slugify = (text) =>
     .replace(/\s+/g, '-')
     .replace(/[^\w-]+/g, '')
     .replace(/--+/g, '-');
+
+export const verifyNigerianPhoneNumber = (number) =>
+  /^([0]{1}|\+?234)([7-9]{1})([0|1]{1})([\d]{1})([\d]{7})$/g.test(number);
+
+export const sanitizePhoneNumber = (number) => {
```

Comment:
Same comment as above but we would just add various variants of valid Nigerian phone numbers and test that they return the right sanitized number. We should test for space, dash, and plus as specified in the function.

Reply:
With this implementation, I think it is better to add the `countryCode` to the function and make it default to Nigeria Country code. That way, the function would easily be reusable for all countries.

```suggestion
export const sanitizePhoneNumber = (number, countryCode = 234) => {
```